packages/document/main-doc/docs/en/guides/basic-features/alias.mdx

```diff
@@ -27,17 +27,19 @@ For example, import the modules from the `src/common/` directory in the `src/App
 │   │   └── base.css
 │   └── utils
 │       └── index.ts
-├── App.tsx
+└── App.tsx
 ```
 
-the code in `src/App.tsx`:
+The code in `src/App.tsx` is as follows:
 
 ```ts
 import utils from '@/src/common/utils';
 import '@/src/common/styles/base.css';
 ```
 
-Modern.js also provides a way to config aliases. Adding the `@common` alias as an example. For TypeScript projects, you only need to configure `compilerOptions.paths` under the project root directory `tsconfig.json` as follows:
+Modern.js also provides a way to customize alias. Taking adding the `@common` alias as an example:
```

Comment:
Taking adding the `@common` alias as an example -> For example,  adding the `@common` alias is as follows

This expression is simpler and easy to read.